RELEASE.md

```diff
@@ -1,28 +1,44 @@
 # SNIRF Release Procedure
 
 This document describes the process for releasing a new version of the SNIRF specification.
-Only someone with write access to the `fNIRS/snirf` repository can complete this procedure.
+This procedure should be completed by a steering committee member in accordance with [governance policy](GOVERNMENT.md).
 
 1. Raise an issue at least one week prior to a planed release date to inform the community that a release is imminent.
-   Ensure that the issue clearly states the exact next proposed version number (e.g., `v1.0`).
-2. Ensure at least one other admin (someone with write access) agrees with the planned release,
-   and confirms that there are no known blocking issues (open issues in the tracker are ok, but they should not be of critical nature). 
-   Approval should be provided as a comment in the issue, not via personal communication.
+   Ensure that the issue clearly states the exact next proposed version number (e.g., `v1.0.0`).
+2. Unanimous approval is required from the steering committee.
+   Ensure all steering committee members have explicitly given their approval as a public github comment in the issue.
 3. Open a pull request that makes the following modifications:
-   1. Modify the version number in the opening lines of the specification to match the version to be released (e.g., `v1.0`).
+   a. Modify the version number in the opening lines of the specification to match the version to be released (e.g., `1.0.0`).
```

Comment:
Github is not parsing this markdown as a (sub-) bullet point, it appears inline in the preview text. Likely whitespace or numeral/character mix issue. This applies to all bullets below.